src/main/archetype/ui.content/src/main/content/jcr_root/conf/__confFolderName__/settings/wcm/policies/.content.xml

```diff
@@ -12,7 +12,7 @@
                         jcr:primaryType="nt:unstructured"
                         jcr:title="${siteName} Default"
                         sling:resourceType="wcm/core/components/policy/policy"
-                        components="[group:${componentGroupName},/apps/${appsFolderName}/components/form/container]"
+                        components="[group:${componentGroupName}.Content,/apps/${appsFolderName}/components/form/container,group:${componentGroupName}.Structure]"
```

Comment:
`group:${componentGroupName}.Content`,`group:${componentGroupName}.Form,`

Reply:
Only the form container should be available for the root repsonsivegrid.